hack/verify-kep-metadata.sh

```diff
@@ -18,17 +18,15 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-export KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
-cd ${KUBE_ROOT}
+TOOL_VERSION=b97ce7d3f1daca6bc79d60952e461df3ffa88a5b
```

Comment:
non-blocking nit: can we create a tag/version at this sha?

Reply:
yes we certainly can!

Reply:
turns out there was already a version, I just hadn't referenced it :)